.gitignore

```diff
@@ -35,4 +35,6 @@ out/build/**
 EnvironmentSimulator/Libraries/esminiJS/build*/*
 support/generated
 pylint_report.txt
-CMakeUserPresets.json
\ No newline at end of file
+CMakeUserPresets.json
```

Comment:
Regarding the three added files to the ignore list:
I googled .DS_Store and now understand that it's a Mac system file, which clearly can be ignored 👍
But the other two:
- CMakeUserPresets.json
- esmini.js
I would suggest they are not ignored. I.e. they should be removed from the list. Would that be OK with you? Or do you have some reasons why they should be ignored?

Reply:
Thanks for clarifications. Indeed CMakeUserPresets.json was already included. My mistake. 
Same for esmini.js - you're completely right, generated files should be ignored.